### ISSUE TITLE
some textures can't be extracted (x64.v0.11.26)

### ISSUE THREAD

**Issue by Tosyk** [Perfare/AssetStudio#297]
Game 'Vertigo Racing'

AssetStudio x64.v0.11.26 can't extract mask textures. But x64.v0.11.16 does it well.
like this one:
![image](https://user-images.githubusercontent.com/24209966/48218520-1a91d100-e39b-11e8-8b60-d6c473760bc2.png)

**Comment by Tosyk** [comment]
Yeah, sorry. I should say that this image exported with FBX file.
firstly I exported FBX with x64.v0.11.26 and I got only basic diffuse texture with some plain color PNG
secondly I used x64.v0.11.16 version for exactly same thing and got +1 texture which is this mask

I do something wrong with newer version?

**Comment by Perfare** [comment]
In version 0.11.16, although this image will be exported but not connected to the fbx file, so 0.11.26 version no longer export it.

**Comment by Tosyk** [comment]
this texture might not be connected with model technicaly but it related to the model and 0.11.16 version  exported it fine. How to reproduce the same behavior on 0.11.26 version(?) because I need these kind of textures along with the model
